<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>局域网聊天工具</string>
  </property>
  <widget class="QPushButton" name="m_ButtonSend">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="m_EditInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Reg">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_Pass">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>用户密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>用户ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_Name">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_ID">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_Port">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Exit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>223</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>离线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Login">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_ButtonExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Text_IP">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10.21.3.103</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_EditRecord">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="m_UserList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
